src/plantuml_generator.rs

```diff
@@ -114,26 +118,28 @@ impl<'a> PlantUmlDefaultGenerator<'a> {
                         .unwrap()
                 })
         };
-        self.str_templates
-            .render(
-                "ent",
-                &SEntity {
-                    pks: columns_render(true, false),
-                    fks: columns_render(false, true),
-                    others: columns_render(false, false),
-                    name: tbl.name.clone(),
-                },
-            )
-            .unwrap()
+        Ok(self.str_templates.render(
+            "ent",
+            &SEntity {
+                pks: columns_render(true, false),
+                fks: columns_render(false, true),
+                others: columns_render(false, false),
+                name: tbl.name.clone(),
+            },
+        )?)
     }
 }
 
 impl<'a> ViewGenerator for PlantUmlDefaultGenerator<'a> {
-    fn generate(&self, sql_erd: SqlERData, opts: &GeneratorConfigOptions) -> String {
+    fn generate(
+        &self,
+        sql_erd: SqlERData,
+        opts: &GeneratorConfigOptions,
+    ) -> Result<String, crate::SqlantError> {
         let entities: Vec<String> = sql_erd
             .tables
             .iter()
-            .map(|tbl| self.entity_render(tbl, opts))
+            .map(|tbl| self.entity_render(tbl, opts).unwrap())
```

Comment:
We can use "?" see message above

Reply:
There are a lot more unwraps & panics which can be removed.
This PR is about adding `Result` to the public APIs.